_episodes/14.md

```diff
@@ -0,0 +1,330 @@
+a Senior Software Engineer at Netflix who created this weeks' 
+[PR of the week](https://github.com/trinodb/trino/pull/1067).
+
+[Release 318](/docs/current/release/release-318.html), 
+introduced this code that supported querying tables from Apache Iceberg in
+Trino. While the code existed, the Iceberg connector code wasn't officially
+released or documented until a little over a year later in 
+[release 341](/docs/current/release/release-341.html) once the connector reached
+maturity. 
+
+## Demo: Creating tables through Iceberg and reading them through Trino
+
+For this weeks' demo, I wanted to play around with the Iceberg Java API directly.
+You also have the option to use Trino, Spark, or other to ingest and query the
+data, but I wanted to use vanilla Iceberg API's to experience the API and
+hopefully solidify my learning of Iceberg concepts in the process. Make sure you
+follow the instructions in the repository if you don't have Docker or Java
+installed.
+
+Let's start up a local Trino coordinator and hive metastore. Clone this 
+repository and navigate to the `iceberg/trino-iceberg-minio` directory. Then
+start up the containers using Docker Compose.
+
+```
+git clone git@github.com:bitsondatadev/trino-getting-started.git
+
+cd iceberg/trino-iceberg-minio
+
+docker-compose up -d
+```
+
+In your favorite IDE, open the files under `iceberg/iceberg-java` into your
+project and run the `IcebergMain` class.
+
+This class creates a logging table if it doesn't exist. Once you run this code,
+you can check to see that the table in Trino exists in the metastore under
+TABLE_PARAMS. But, if run `SHOW TABLES IN iceberg.logging;` you'll notice that
+the table doesn't show up due to [the issue we discussed above](https://github.com/trinodb/trino/issues/1592).
+
+Let's update the TABLE_PARAMS entry in the metastore db and then query the table
+again.
+
+## Question of the week: Why do we still depend on the Hive metastore if metadata for Iceberg saves to the filesystem?
+
+We kept the metastore as many tests run around using the metastore that exist
+for the Hive connector and we want to give the Iceberg connector ample time to
+mature before we migrate entirely away from the metastore. We also wanted to 
+make the metastore the initial method of use in Iceberg that got developed as
+most developers initially would be migrating from their existing Hive catalog
+and we wanted this transition to use existing tested components.
+
+Currently, the metastore isn't used the same way as in Hive. Trino stores a
+top-level directory that points to the metadata manifest file location and other
+statistics around the table in the TABLE_PARAMS table of the metastore. There
+is a [pull request created by Jack Ye](https://github.com/trinodb/trino/pull/6977)
+to migrate away from using the metastore, but in the meantime, we must still
+use a separate metastore.
+
+## Events, news, and various links
```

Comment:
Above this add section about Iceberg tipping ... its mostly written up in the Gdoc